app/src/hooks/useGoogleOneTapLogin.ts

```diff
@@ -23,12 +25,32 @@ interface CredentialResponse {
   client_id: string;
 }
 
+export const shouldShowOneTapPrompt = () => {
+  if (
+    window.location.href.includes(PATHS.AUTH.DEKSTOP_SIGN_IN.RELATIVE) ||
+    window.location.href.includes(PATHS.AUTH.EMAIL_ACTION.RELATIVE) ||
+    window.location.href.includes(PATHS.AUTH.EMAIL_LINK_SIGNIN.RELATIVE)
+  ) {
+    return false;
+  }
+  return true;
+};
+
```

Comment:
we should not export anything else except the hook from the hook file.
We can make the hook can return this util function and then use it in `DashboardLayout`